### ISSUE TITLE
Wrong comment above property

### ISSUE THREAD

**Issue by joshglick** [guillermomuntaner/GMImagePicker#21]
Looks like a copy and paste issue of a comment from below.

From the file:
```
/**
 *  Determines whether or not the number of assets is shown in the Album list.
 *  The number of assets is visible by default.
 */
@property (nonatomic, strong) NSArray* customSmartCollections;

...

/**
 *  Determines whether or not the number of assets is shown in the Album list.
 *  The number of assets is visible by default.
 */
@property (nonatomic, assign) BOOL displayAlbumsNumberOfAssets;
```

The first one should be changed to reflect the property, I'm not super familiar or I'd submit a pull request. Maybe I can try to help later this week.

**Comment by guillermomuntaner** [comment]
Indeed, copy paste issue. Thanks for the report.

**Comment by joshglick** [comment]
No problem!

**Comment by guillermomuntaner** [comment]
This has been corrected by @albertbori in merge #14
